.flowconfig

```diff
@@ -40,7 +40,6 @@
 .*/node_modules/shelljs/.*
 .*/node_modules/table.*
 .*/node_modules/tar.*
-.*/node_modules/tmp.*
```

Comment:
This is because we use this `tmp` module in `packages/product-json-to-xlsx/src/writer.js` and when it is ignored by `.flowconfig` I get this flow error:
`Cannot resolve module tmp.`